api/api.go

```diff
@@ -8,8 +8,8 @@ import (
 type ClientService interface {
 	ListServices(inst flux.InstanceID, namespace string) ([]flux.ServiceStatus, error)
 	ListImages(flux.InstanceID, flux.ServiceSpec) ([]flux.ImageStatus, error)
-	PostRelease(flux.InstanceID, flux.ReleaseJobSpec) (flux.ReleaseID, error)
-	GetRelease(flux.InstanceID, flux.ReleaseID) (flux.ReleaseJob, error)
+	PostRelease(flux.InstanceID, flux.ReleaseJobParams) (flux.JobID, error)
+	GetRelease(flux.InstanceID, flux.JobID) (flux.Job, error)
```

Comment:
A way to avoid this might be to keep the ReleaseJob struct in the API, and transplant all the values from Job into it (since they are the same, really) when serving it. After all, it's `GetRelease`, not `GetJob`.